tests/utils/editor_recommendation_test.py

```diff
@@ -0,0 +1,47 @@
+import pandas as pd
+
+from data_science_pipeline.utils.editor_recommendation import (
+    get_author_ids_of_given_version_of_paper
+)
+
+MANUSCRIPT_VERSION_1 = "match_version_id_1"
+
+AUTHOR_ID_LIST_1 = [
+    "match_author_id1",
+    "match_author_id2"
+]
+
+MANUSCRIPT_VERSION_2 = "unmatch_version_id_1"
+
+AUTHOR_ID_LIST_2 = [
+    "unmatch_author_id1"
+]
+
+MANUSCRIPT_DICT = {
+  "version_id": {
+    "0": MANUSCRIPT_VERSION_1,
+    "1": MANUSCRIPT_VERSION_2
+  },
+  "author_person_ids": {
+    "0": AUTHOR_ID_LIST_1,
+    "1": AUTHOR_ID_LIST_2
+  }
+}
+
+MANUSCRIPT_DF = pd.DataFrame(MANUSCRIPT_DICT)
+
+print(MANUSCRIPT_DF)
+
+
+class TestGetAuthorIdsOfGivenVersionOfPaper:
+    def test_should_return_author_ids_for_matching_manuscript_version(self):
+        assert get_author_ids_of_given_version_of_paper(
+            MANUSCRIPT_DF,
+            MANUSCRIPT_VERSION_1
+        ) == AUTHOR_ID_LIST_1
+
+    def test_should_not_return_author_ids_for_unmatching_manuscript_version(self):
```

Comment:
This test almost doesn't seem necessary. If we are returning exactly the expected author person ids, then we are not returning the other author person ids.

The test would then mainly add though that we are testing it with another version id. This would prove that the function isn't just picking the first row. So there is that value. One shortcut I've taken in the past was to just have another third row as the first row, i.e. the function would have to pick the middle row. Or use parameterize, or another test function like this. I would just suggest to rename it in that case. Not quite sure to what, maybe "should return author ids for other manuscript version id".